src/app/containers/NewStoryPromo/PromoOrganisms/TopStoriesPromo/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { arrayOf, shape } from 'prop-types';
+import { storyItem } from '#models/propTypes/storyItem';
+import pathOr from 'ramda/src/pathOr';
+import { C_GREY_2 } from '#app/legacy/psammead-styles/src/colours';
+import Promo from '../../Promo';
+import { TopStoriesPromoWrapper } from './index.styles';
+import TopStoriesItem from './TopStoriesMolecules/TopStoriesItem';
+import TopStoriesList from './TopStoriesMolecules/TopStoriesList';
+
+const TopStoriesPromo = ({ content }) => {
+  const { script, service, dir, translations } = useContext(ServiceContext);
+  const title = pathOr('Top Stories', ['topStoriesTitle'], translations);
+  const hasSingleContent = content.length === 1;
+  const LABEL_ID = 'top-stories-heading';
+
+  return (
+    <Promo Wrapper={TopStoriesPromoWrapper}>
+      <Promo.SectionLabel
+        script={script}
+        service={service}
+        dir={dir}
```

Comment:
I'd recommend just giving the components these props at source - means they don't need to be provided every time the component is used.  Could look something like this: https://github.com/bbc/simorgh/blob/latest/src/app/components/Promo/index.jsx#L67

Reply:
Done

Reply:
👍 nice!  That current implementation does require all the components to be a child of the <Promo> component, which is probably a reasonable restriction

But as an alternative, this could be `ServerContext.Consumer` instead:

https://github.com/bbc/simorgh/pull/10165/commits/119a746203f18e1a0f8ad38dc3ec2236f593b2c6#diff-3a4ede4b4a097ef48b30ada95e57aac216160111b4e71f015d939c13d93e2a7bR32